src/runtime/basic-runtime.js

```diff
@@ -244,6 +245,13 @@ export class BasicRuntime {
   setupButtons() {
     return this.configured_(false).then((runtime) => runtime.setupButtons());
   }
+
+  // Process result from checkentitlements view
```

Comment:
Could you make this a JSDoc comment? (/** ...)

Reply:
Done